libs/gltfio/src/extended/AssetLoaderExtended.cpp

```diff
@@ -527,11 +527,11 @@ bool AssetLoaderExtended::createPrimitive(Input* input, Output* out,
 
     if (!mCgltfBuffersLoaded) {
         mCgltfBuffersLoaded = utility::loadCgltfBuffers(gltf, mGltfPath.c_str(), mUriDataCache);
+        utility::decodeMeshoptCompression(gltf);
```

Comment:
What if this function is called with a different `input->gltf`?

Reply:
that would be true for line 529 too right? this function is called from only one place, at-least with the current implementation,  `FAssetLoader::createPrimitives` which passes the `gltf` from root asset.  May be do both `loadCgltfBuffers` & `decodeMeshoptCompression` at the calling site (FAssetLoader::createPrimitives) similar to `ResourceLoader::loadResources`?

Reply:
yeah, I need to look at this closer I think.  For now, this patch is good to go.